storage/digital_ocean_spaces.py

```diff
@@ -83,3 +83,26 @@ def upload_content(
         self._client.upload_fileobj(
             f, self._bucket, file_key, ExtraArgs={"ACL": permission}
         )
+
+    def upload_zip(
+        self,
+        file_key: str,
+        content_to_be_uploaded: BytesIO,
+        permission: str = "public-read",
+    ) -> None:
+        logging.debug(f"Uploading {file_key}")
+        self._client.upload_fileobj(
+            content_to_be_uploaded, self._bucket, file_key, ExtraArgs={"ACL": permission}
+        )
+
+    def copy_file(self, source_file_key: str, destination_file_key: str) -> None:
+        logging.debug(f"Copying {source_file_key} to {destination_file_key}")
+        self._client.copy_object(
+            Bucket=self._bucket,
+            CopySource={'Bucket': self._bucket, 'Key': source_file_key},
+            Key=destination_file_key
+        )
+
+    def delete_file(self, file_key: str) -> None:
+        logging.debug(f"Deletando {file_key}")
```

Comment:
Só pra seguir o padrão do projeto
```suggestion
        logging.debug(f"Deleting {file_key}")
```

Reply:
irei mudar

Reply:
Implementado!